### PR TITLE
Merging Color MEI code changes

### PR DESCRIPTION
**Pull request by Synicix**
Please note that one of the hacks is the skip_duplicates for stimulus assignment to either train, val or test. We had to get around that to get neurodata to process even though if it means that the train val and test sets will be screwed up.

### CODE REVIEW

File: neuro_data/static_images/data_schemas.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     'stimulus.ColorFrameProjector': ('image_id', 'image_class'),
 }
 
-IMAGE_CLASSES = 'image_class in ("imagenet", "imagenet_v2_gray", "imagenet_v2_rgb")' # all valid natural image classes
+IMAGE_CLASSES = 'image_class in ("imagenet", "imagenet_v2_gray", "imagenet_v2_rgb", "imagenet_v2_rgb_g_b_channels_separately_joined")' # all valid natural image classes
```
Comment (fabiansinz) [inline neuro_data/static_images/data_schemas.py:36]: @Synicix are these all classes or do we need to update this list with Donnie's latest ones?

File: neuro_data/static_images/data_schemas.py [hunk 2]
```diff
@@ -141,7 +141,7 @@ def fill(self, scan_key):
             print('Static images were not shown for this scan')
 
         # Get all image ids in this scan
-        all_frames = frame_table * stimulus.Trial & scan_key & IMAGE_CLASSES
+        all_frames = frame_table * stimulus.Trial & scan_key
```
Comment (fabiansinz) [inline neuro_data/static_images/data_schemas.py:144]: Why are `IMAGE_CLASSES` gone here?

File: neuro_data/static_images/data_schemas.py [hunk 3]
```diff
@@ -245,7 +245,7 @@ def make(self, key):
 
                 # deal with ImageNet frames first
                 log.info('Inserting assignment from ImageNetSplit')
-                targets = StaticScan * frame_table * ImageNetSplit & (stimulus.Trial & key) & IMAGE_CLASSES
+                targets = StaticScan * frame_table * ImageNetSplit & (stimulus.Trial & key)
```
Comment (fabiansinz) [inline neuro_data/static_images/data_schemas.py:248]: ... and here

**Comment by Synicix** [comment]
Donnie kept creating new ones to I ended up trashing the entire IMAGE_CLASSES restriction hences the changes above.

**Comment by fabiansinz** [comment]
I don't understand the reasoning: You are saying because there are more classes you get rid of all constraints? This would mean that other things like MEIs would end up in this dataset as well.

**Comment by Synicix** [comment]
The way that Donnie was doing it in the past was for each new stimulus Katrin and him wanted to test they create an entirely new class for it. I can't call the exact reason for it, but the consequence of that was that each time I ran the neurodata package for them targets would be empty cause it is not part of the one of the specified classes. After changing it a couple of time I just got rid of the entire thing so I don't have to change the code each time they add a new class.

**Pull request closed without merging**